src/protected_flash.rs

```diff
@@ -1329,9 +1348,11 @@ where
         cursor.write_all(&u32::from(self.rot_keys_status).to_le_bytes())?;
         cursor.write_all(&self.vendor_usage.into().to_le_bytes())?;
 
-        let debug_settings: [u32; 2] = Into::<DebugSecurityPolicies>::into(self.debug_settings).into();
-        cursor.write_all(&debug_settings[0].to_le_bytes())?;
-        cursor.write_all(&debug_settings[1].to_le_bytes())?;
+        // Debug settings.
+        // Should never externally set the debug policy bits in customer page.
+        // These bits should only be changed from secure API within firmware.
+        cursor.write_all(&0u32.to_le_bytes())?;
+        cursor.write_all(&0u32.to_le_bytes())?;
```

Comment:
I made non pub and added the getter.  Keeping writing just zeros, since converting from the debug_settings directly will always set the inverse bits (i.e. it can't be nonzero currently without a large refactor).  I'm happy to leave it as is.